SECP-service/src/main/java/com/visucius/secp/Controllers/GroupController.java

```diff
@@ -0,0 +1,136 @@
+package com.visucius.secp.Controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.visucius.secp.DTO.GroupCreationRequest;
+import com.visucius.secp.daos.GroupDAO;
+import com.visucius.secp.daos.RolesDAO;
+import com.visucius.secp.daos.UserDAO;
+import com.visucius.secp.models.Group;
+import com.visucius.secp.models.Role;
+import com.visucius.secp.models.User;
+import com.visucius.secp.util.InputValidator;
+import com.visucius.secp.util.JsonUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class GroupController {
+
+    private static final Logger LOG = LoggerFactory.getLogger(GroupController.class);
+    public static final int MINIMUM_AMOUNT_OF_USERS = 2;
+    public static final int MAXIMUM_AMOUNT_OF_USERS = 50;
+
+    private GroupDAO groupRepository;
+    private UserDAO userRepository;
+    private RolesDAO rolesRepository;
+
+    public GroupController(GroupDAO groupRepository, UserDAO userRepository, RolesDAO rolesRepository)
+    {
+        this.groupRepository = groupRepository;
+        this.userRepository = userRepository;
+        this.rolesRepository = rolesRepository;
+    }
+
+
+    public Response createGroup(GroupCreationRequest request)
+    {
+        List<String> errors = validateCreationRequest(request);
+
+        if(!errors.isEmpty())
+        {
+            try
```

Comment:
what's the point of this try catch block. Can't you just throw an error once you find an issue with the json. Take a look at the way i did it for the AdminResource.